### ISSUE TITLE
Proposal: Ship stickers to first-time PR contributors in StackStorm repos

### ISSUE THREAD

**Issue by nmaludy** [StackStorm/discussions#43]
### Background

Would like to get the community more engaged in the StackStorm development lifecycle.

We've seen several new community members submit drive-by PRs and want to reward them and encourage them to come back to develop the packs further.

### Proposal

One thought around this is to reward people for their code contributions with some StackStorm stickers.

We could buy stickers in bulk from a place like Sticker Mule (https://www.stickermule.com/) and buy some Forever stamps then ship them upon first PR.

I would take responsibility for managing this and mailing the envelopes.

### Cost breakdown

- Stamps: $0.55 per stamp for domestic US $1.20 per stamp for international
  https://store.usps.com/store/results/stamps-forever-55-cents/_/N-31vwta
  https://www.usps.com/international/first-class-mail-international.htm
- Stickers: $295 for 1,000 3x3 stickers ($0.295 / sticker)
- Envelopes: $7.88 for 500 envelopes ($0.015 / envelope)

Total (US): $0.55 + 3*$0.295 + $0.015 = $1.45
Total (International): $1.20 + 3*$0.295 + $0.015 = $2.10

Total investment for 1,000 stickers + envelopes + stamps
- US = (1000*$0.55) + $295 + $15.76 = $860.76
- International = (1000*$1.20) + $295 + $15.76 = $1,510.70

**Comment by arm4b** [comment]
+100!

Between all the expenses (#36), that's one of the most inspiring things where we could spend `$` donated to StackStrom CommunityBridge https://funding.communitybridge.org/projects/stackstorm

**Comment by arm4b** [comment]
@nmaludy 1K stickers could be a bit big, how the numbers might look like for 200? So we start with something small as a pilot project.

**Comment by nmaludy** [comment]
StickerMule does pretty aggressive pricing for bulk buys, so 200 stickers is $112 ($0.55 per sticker) opposed to $296 for 1,000 ($0.296 per sticker).

Here is the cost breakdown for 200:


    US = (200*$0.55, postage) + ($112, stickers) + ($15.76, envelopes) = $237.76
    International = (200*$1.20, postage) + ($112. stickers) + ($15.76, envelopes) = $367.76


That is definitely a lot less money because shipping / postage is the predominant cost here. This gives us a minimum and maximum bounds on the cost of the project depending on the make up of US and International users.

**Comment by arm4b** [comment]
That looks good!
It's now something we can start with, considering current https://funding.communitybridge.org/projects/stackstorm funds.

**Comment by arm4b** [comment]
I guess another interesting question is how do we promote the program on a constant basis?

Apart of Blog Post announcement, Twitter, LinkedIn, Email Newsletter, maybe something else to get creative? I don't know, kind of on-going automation around Github PR Bots, Slack welcome message Bots? Definitely a lot to explore here.

@nmaludy WDYT? @StackStorm/tsc any ideas?

**Comment by nmaludy** [comment]
Yeah maybe a GitHub PR bot when a new PR is created post a message back:

```
Thanks for submitting your issue, we really appreciate contributions from community members!

Since the community is so important to us, we reward first time PR contributors with stickers, shipped directly to them.

If you would like to try fixing this issue yourself by submitting a PR we would be happy to review and send you some totally amazing StackStorm stickers in return!

Otherwise, we'll be around to check on this issue and get back to you shortly.

Thanks,
StackStorm Exchange Team
```

**Comment by Kami** [comment]
I still have some original StackStorm logo and robot stickers so if anyone inside Europe wants some, I'm happy to mail them out.

**Comment by nmaludy** [comment]
@Kami Can you shoot me an email at: nick@stackstorm.com ? I have one person in Europe that i need to send stickers to, but the post office here didn't have any international stamps.

Thanks!
